### ISSUE TITLE
Can't switch Themes

### ISSUE THREAD

**Issue by arielelkin** [jonschlinkert/sublime-monokai-extended#81]
ST Build 3126

1. Installed `sublime-markdown-extended` via Package Control
1. Installed `sublime-monokai-extended` via Package Control

I'm expecting my menu to be like this:
`Preferences -> Color Scheme -> User -> Monokai Extended.`

But instead I have 

`Preferences -> Color Scheme -> Monokai Extended.`

Maybe similar to https://github.com/jonschlinkert/sublime-monokai-extended/issues/45, but I installed and uninstalled MarkdownEditing (via Package Manager) just to be sure, and issue persists.

**Comment by arielelkin** [comment]
bump :-)

**Comment by jonschlinkert** [comment]
This is what I see

![image](https://cloud.githubusercontent.com/assets/383994/22956497/63f1c910-f2f0-11e6-8c36-da62fadc9c94.png)

**Comment by arielelkin** [comment]
this is what I see:

![image](https://cloud.githubusercontent.com/assets/1756909/22956662/afc660d4-f323-11e6-84e7-f3f17fa9c266.png)

**Comment by jonschlinkert** [comment]
Ah, (I think) I know what's happening. My docs are wrong. My guess is that it only shows up in `user` on mine because I actually copied those files into that directory for development (I'd facepalm lol, but Idk if it's obvious or not, since it's been that way since I first started using ST). But for you, it looks correct, and fwiw I also have the same preferences as you (in addition to the `user` option).
